source/libsodium/crypto_core_salsa208.d

```diff
@@ -31,3 +31,4 @@ int crypto_core_salsa208 (
     const(ubyte)* in_,
     const(ubyte)* k,
     const(ubyte)* c);
+
```

Comment:
Neither does the double newline at the end of the file

Reply:
It really matters?

Reply:
Yes

Reply:
Looks ike it is because upstream `.h` file contains some empty lines and leaved unparsed by `dstep` `#endif` at its end.

Reply:
(That is, this is not the result of the negligence of this script.)